NEWS.md

```diff
@@ -1,6 +1,31 @@
 MathOptInterface (MOI) release notes
 ====================================
 
+v0.9.0 (May 2?, 2019)
+---------------------
+
+- Support for Julia v0.6 and v0.7 are dropped (#714, #717).
+- Add support for indicator constraints (#709).
+- Add `RawParameter` attribute (#733).
+- The field of `ObjectiveValue` was renamed to `result_index` (#729).
+- The `_constant` and `Utilities.getconstant` function are renamed to `constant`
+  (#723).
+- Add `REDUCTION_CERTIFICATE` result status (#734).
+- Add abstract matrix sets (#731).
+- Testing improvements:
```

Comment:
Consistently use phrases or full sentences, e.g.,
either:
- Updated testing guideline
- Added quadratic tests

or:

- The testing guideline was updated.
- Quadratic tests were added.